osrm/core.py

```diff
@@ -143,15 +154,14 @@ def simple_route(coord_origin, coord_dest, coord_intermediate=None,
     geometry : str, optional
         Format in which decode the geometry, either "polyline" (ie. not decoded),
         "geojson", "WKT" or "WKB" (default: "polyline").
+    annotations : str, optional
+    continue_straight : str, optional
```

Comment:
It looks like your are not using the continue_straight option when building the url ?

Reply:
Good catch, fixed.